### PR TITLE
[spec] nnfw now packaged only for tizen

### PR DESCRIPTION
**Pull request by kparichay**
nnfw package is now created and corresponding tests performed
only for tizen in the spec file.

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -101,11 +104,6 @@ BuildRequires: ssat
 # For ORC (Oil Runtime Compiler)
 BuildRequires:	pkgconfig(orc-0.4)
 
-%if %{with tizen}
```
Comment (dongju-chae) [inline packaging/nnstreamer.spec]: is this not required now?

Comment (kparichay) [reply packaging/nnstreamer.spec]: This `if %{with tizen}` block added `BuildRequires` for tizen sensor framework.
This has been moved to line 82 into the existing `if %{with tizen}` block

Comment (dongju-chae) [reply packaging/nnstreamer.spec]: Ah.. I missed line 71.

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -455,6 +446,13 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %files -n nnstreamer-tizen-internal-capi-devel
 %{_includedir}/nnstreamer/nnstreamer-tizen-internal.h
 
+%if 0%{?enable_nnfw_r}
+%files nnfw
+%manifest nnstreamer.manifest
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_nnfw.so
```
Comment (wooksong) [inline packaging/nnstreamer.spec]: Instead of hard-coded 'lib', it is better to use %{_libdir}.

Comment (myungjoo) [reply packaging/nnstreamer.spec]: No.
For application/package specific paths, it is /usr/lib/NAME/whatsoever, not %{_libdir}/NAME/whatsoever.

**Pull request merged**